packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/__tests__/quanticBreadcrumbManager.test.js

```diff
@@ -5,6 +5,10 @@ import QuanticBreadcrumbManager from 'c/quanticBreadcrumbManager';
 import {createElement} from 'lwc';
 import * as mockHeadlessLoader from 'c/quanticHeadlessLoader';
 
+jest.mock('@salesforce/label/c.quantic_Colon', () => ({default: ':'}), {
```

Comment:
Wait why do we have a label for a colon?

Reply:
yeah why in case we want to translate it x)

Reply:
it was added while ago in 2021 and it's yes for translation, cause in french there is a space before the colon while in english there is no space.